libs/template/templates/default-python/template/{{.project_name}}/databricks.yml.tmpl

```diff
@@ -0,0 +1,32 @@
+bundle:
+  name: com.mycompany.{{.project_name}}
+
+include:
+  - res/*.yml
+
+artifacts:
+    {{.project_name}}:
+      type: whl
+      path: "."
+      build: "python3 setup.py bdist_wheel"
+
+environments:
```

Comment:
We should start using `targets` instead of `environments`

Reply:
+1, local copy already has `targets`.